common.js

```diff
@@ -1943,6 +1943,64 @@ const allSoloCars = {
       '2012': ['gs']
     },
   },
+  'Opel': {
+      '1900': ['fsp'],
+      'Ascona': {
+        '1971': ['dp'],
+        '1972': ['dp'],
+        '1973': ['dp'],
+        '1974': ['dp'],
+        '1975': ['dp']
+      },
+      'Ascona SportWagon (1900cc)': {
+        '1971': ['dp'],
+        '1972': ['dp'],
+        '1973': ['dp'],
+        '1974': ['dp'],
+        '1975': ['dp']
+      },
+      'Manta': ['fsp'],
```

Comment:
This structure does not match the rest of object. The structure in the rest of the object is
```
make: {
  model: { 
    year/all: [class1, class2]
  }
}
```

The structure you provided here is
```
make: {
  model: [class1, class2]
}
```

Please adjust this entry. Thanks!